src/Microsoft.Dnx.Runtime/NuGet/Utility/PathUtility.cs

```diff
@@ -72,7 +73,72 @@ public static string GetRelativePath(string path1, string path2)
         /// </summary>
         public static string GetRelativePath(string path1, string path2, char separator)
         {
-            return UriUtility.GetPath(GetRelativeUri(path1, path2), separator);
+            if (path1 == null)
```

Comment:
also all the tests are using absolute paths - what if a relative path is passed?